import_classifiers.R

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/env Rscript 
+
+# Import pre-trained classifiers for a provided list of datasets
+
+option_list = list(
+    make_option(
+            c("-f", "--config-file"),
+            action = "store",
+            default = NA,
+            type = 'character',
+            help = "Config file in .yaml format"
+        ),
+    make_option(
+            c("-t", "--tool"),
+            action = "store",
+            default = NA,
+            type = 'character',
+            help = "Which tool's classifiers should be imported?"
+        ),
+    make_option(
+            c("-c", "--classifiers-output-dir"),
+            action = "store_true",
+            default = NA,
+            type = 'character',
+            help = "Path for directory storing imported classifiers"
+    )
+)
+
+opt = wsc_parse_args(option_list, mandatory = c("tool", "classifiers_output_dir"))
+# import dependencies
+suppressPackageStartupMessages(require(optparse))
+suppressPackageStartupMessages(require(workflowscriptscommon))
+suppressPackageStartupMessages(require(R.utils))
+suppressPackageStartupMessages(require(yaml))
```

Comment:
Probably move the yaml load to the condition below- you don't need it if you don't need to load the yaml